src/plugins/intel_cpu/tests/unit/streams_info_table_test.cpp

```diff
@@ -15,6 +15,96 @@ using namespace ov;
 
 namespace {
 
```

Comment:
can we also add test for property here https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp#L105 ?

They are generic tests which can be enabled for all plugins

Reply:
UseHTTests is CPU plugin specific test case based on proc_type_table. 

Do you mean behavior test case for ov::hint::use_hyper_threading in src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/ov_plugin/properties_tests.cpp ?

Reply:
> Do you mean behavior test case for ov::hint::use_hyper_threading

Yes

Reply:
Updated